src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,74 @@
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        System.out.println("--------------Test create car----------------");
+        Manufacturer manufacturer = manufacturerService.get(9L);
+        Driver driverFirst = driverService.get(17L);
+        Driver driverSecond = driverService.get(18L);
+        Driver driverThird = driverService.get(15L);
+        Car car = new Car();
+        car.setManufacturer(manufacturer);
+        car.setDrivers(new ArrayList<>(Arrays.asList(driverFirst, driverSecond, driverThird)));
+        car.setModel("Nubira");
+        System.out.println(car);
+        CarService carService
+                = (CarService) injector.getInstance(CarService.class);
+        carService.create(car);
+        System.out.println(car);
+        System.out.println("--------------Test get car by----------------");
+        System.out.println(carService.get(car.getId()));
+        System.out.println("--------------Test update car----------------");
+        Car carFirst = new Car();
+        carFirst.setModel("Corolla");
+        Manufacturer manufacturerFirst = new Manufacturer();
+        manufacturerFirst.setName("Toyota");
+        manufacturerFirst.setCountry("Japan");
+        carFirst.setManufacturer(
+                manufacturerService.get(
+                        manufacturerService.create(manufacturerFirst).getId()));
+        carFirst.setDrivers(new ArrayList<>(Arrays.asList(driverService.get(17L),
+                driverService.get(18L),
+                driverService.get(15L),
+                driverService.get(4L))));
+        System.out.println(carFirst);
+        carService.create(carFirst);
+        carService.get(carFirst.getId());
+        carFirst.setModel("Brinson");
+        List<Driver> drivers = carFirst.getDrivers();
+        drivers.add(driverService.get(5L));
+        carFirst.setDrivers(drivers);
+        carService.update(car);
+        System.out.println("--------------Test get all cars----------------");
+        carService.getAll().forEach(System.out::println);
+        System.out.println("--------------Test get all cars by driver----------------");
+        System.out.println(carService.getAllByDriver(4L));
+        System.out.println("--------------Test delete car----------------");
+        carService.delete(carFirst.getId());
+        carService.getAll().forEach(System.out::println);
+        System.out.println("--------------Test add new driver to car----------------");
+        carService.getAll().forEach(System.out::println);
+        car.getDrivers().add(driverService.get(7L));
+        carService.getAll().forEach(System.out::println);
+        System.out.println("--------------Test remove driver to car----------------");
+        System.out.println(car.getDrivers().remove(driverService.get(7L)));
+        carService.getAll().forEach(System.out::println);
```

Comment:
it is better to add and remove drivers in car not by using remove and add methods from List.
You have addDriverToCar and removeDriverFromCar methods in CarService. So lets use them and test how they work.

Reply:
you are right 100%! I am stupid)